[35mDEBUG   [0m src.steps.common:common.py:18 Running fixture setup: start_1_node
[32mINFO    [0m src.node.status_node:status_node.py:39 Starting node with command: ['./status-cli', 'serve', '-n', 'first_node', '-p', '29639']
[35mDEBUG   [0m src.libs.common:common.py:15 Sleeping for 2 seconds
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - 2025-02-24T00:31:15.469Z	INFO	status-cli/serve.go:34	Running serve command, with:
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - -name first_node
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - -add
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - -port 29639
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - -interactive false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - -light false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - -api-modules waku,wakuext,wakuv2,permissions,eth
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - -telemetry-server-url
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - -debug false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - -fleet status.staging
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - -help false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - 
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - 2025-02-24T00:31:15.469Z	INFO	first_node	status-cli/util.go:55	starting messenger
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - INFO [02-24|00:31:17.716] Starting peer-to-peer node               instance=StatusIM/linux-amd64/go1.22.3
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - INFO [02-24|00:31:17.722] New local node record                    seq=1,740,357,077,720 id=4923ce10c8b49527 ip=127.0.0.1 udp=0 tcp=39263
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - INFO [02-24|00:31:17.722] Started P2P networking                   self="enode://1d010dcb0e007d353449818c337ef3af44a9bea6bc7897361b9ddce5225e9c5c375325f4de83a82e240d4509bb8847e714b422249bd1b1594cae79b2f170e97e@127.0.0.1:39263?discport=0"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - ERROR[02-24|00:31:17.722] Unavailable modules in HTTP API list     unavailable=[wakuv2] available="[admin debug web3 eth rpcstats appmetrics appgeneral peer personal status pending ens communitytokens stickers updates settings accounts multiaccounts browsers permissions mailservers gif chat wallet waku wakuext localnotifications]"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - INFO [02-24|00:31:17.723] HTTP server started                      endpoint=127.0.0.1:29639 auth=false prefix= cors= vhosts=
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - INFO [02-24|00:31:17.750] New local node record                    seq=1,740,357,077,728 id=cd1aec2ad4860f22 ip=10.1.0.33 udp=0 tcp=39573
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - INFO [02-24|00:31:17.751] New local node record                    seq=1,740,357,077,729 id=cd1aec2ad4860f22 ip=10.1.0.33 udp=40736 tcp=39573
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - INFO [02-24|00:31:17.754] New local node record                    seq=1,740,357,077,730 id=cd1aec2ad4860f22 ip=10.1.0.33 udp=40736 tcp=39573
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - 2025-02-24T00:31:18.187Z	INFO	first_node	status-cli/util.go:68	account created, key UID: 0x78d383f404ac29d697905a2f8234f1d048995730b7bb26cee5bea42dcf64a769
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - INFO [02-24|00:31:18.188] New local node record                    seq=1,740,357,077,731 id=cd1aec2ad4860f22 ip=10.1.0.33 udp=40736 tcp=39573
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - 2025-02-24T00:31:18.216Z	INFO	first_node	status-cli/util.go:93	messenger started, public key: 0x040b08668c41af3acb9358cd7b7baec138f5ddac48f7024e8c343d346c1bd17adc07d483bc3c3e7f15c535f75dc4e2fdfd502faea8465075850ab782313f5b05ed
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - INFO [02-24|00:31:22.741] New local node record                    seq=1,740,357,077,732 id=cd1aec2ad4860f22 ip=172.214.173.185 udp=40736 tcp=10186
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - 2025-02-24T00:31:23.217Z	INFO	first_node	status-cli/serve.go:107	retrieve messages...
[32mINFO    [0m src.node.status_node:status_node.py:52 Serving node with command: ['./status-cli', 'serve-account', '-n', '0x02cbc80d3f2c70de89bc14df08cd90af4d038a1ac46ca994a1c1dbafeff4bd46b2_6130', '-kid', '0xd8d0a6ec6b7198cb0217115fd766fab8ff146a351ef79446c2dec6d7dbf7a007']
[35mDEBUG   [0m src.libs.common:common.py:15 Sleeping for 2 seconds
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CBC80D3F2C70DE89BC14DF08CD90AF4D038A1AC46CA994A1C1DBAFEFF4BD46B2_6130 - 2025-02-24T00:31:23.492Z	INFO	status-cli/serve.go:34	Running serve-account command, with:
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CBC80D3F2C70DE89BC14DF08CD90AF4D038A1AC46CA994A1C1DBAFEFF4BD46B2_6130 - -name 0x02cbc80d3f2c70de89bc14df08cd90af4d038a1ac46ca994a1c1dbafeff4bd46b2_6130
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CBC80D3F2C70DE89BC14DF08CD90AF4D038A1AC46CA994A1C1DBAFEFF4BD46B2_6130 - -key-uid 0xd8d0a6ec6b7198cb0217115fd766fab8ff146a351ef79446c2dec6d7dbf7a007
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CBC80D3F2C70DE89BC14DF08CD90AF4D038A1AC46CA994A1C1DBAFEFF4BD46B2_6130 - -interactive false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CBC80D3F2C70DE89BC14DF08CD90AF4D038A1AC46CA994A1C1DBAFEFF4BD46B2_6130 - -add
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CBC80D3F2C70DE89BC14DF08CD90AF4D038A1AC46CA994A1C1DBAFEFF4BD46B2_6130 - -debug false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CBC80D3F2C70DE89BC14DF08CD90AF4D038A1AC46CA994A1C1DBAFEFF4BD46B2_6130 - -help false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CBC80D3F2C70DE89BC14DF08CD90AF4D038A1AC46CA994A1C1DBAFEFF4BD46B2_6130 - 
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CBC80D3F2C70DE89BC14DF08CD90AF4D038A1AC46CA994A1C1DBAFEFF4BD46B2_6130 - 2025-02-24T00:31:23.493Z	INFO	0x02cbc80d3f2c70de89bc14df08cd90af4d038a1ac46ca994a1c1dbafeff4bd46b2_6130	status-cli/util.go:55	starting messenger
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CBC80D3F2C70DE89BC14DF08CD90AF4D038A1AC46CA994A1C1DBAFEFF4BD46B2_6130 - INFO [02-24|00:31:23.533] Starting peer-to-peer node               instance=StatusIM/linux-amd64/go1.22.3
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CBC80D3F2C70DE89BC14DF08CD90AF4D038A1AC46CA994A1C1DBAFEFF4BD46B2_6130 - ERROR[02-24|00:31:23.542] Unavailable modules in HTTP API list     unavailable=[wakuv2] available="[admin debug web3 eth rpcstats appmetrics appgeneral peer personal status pending ens communitytokens stickers updates settings accounts multiaccounts browsers permissions mailservers gif chat wallet waku wakuext localnotifications]"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CBC80D3F2C70DE89BC14DF08CD90AF4D038A1AC46CA994A1C1DBAFEFF4BD46B2_6130 - INFO [02-24|00:31:23.542] New local node record                    seq=1,737,419,564,795 id=21e155c21669159a ip=127.0.0.1 udp=0 tcp=39827
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CBC80D3F2C70DE89BC14DF08CD90AF4D038A1AC46CA994A1C1DBAFEFF4BD46B2_6130 - INFO [02-24|00:31:23.542] Started P2P networking                   self="enode://3adb140cde1eb80c72ac2a83452dabbd2d236498950c66e2194aba96737679facd1c9dc1ec9a7a2052df0b0b89ad1b038868776452403d573e8f3551b0e26fbc@127.0.0.1:39827?discport=0"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CBC80D3F2C70DE89BC14DF08CD90AF4D038A1AC46CA994A1C1DBAFEFF4BD46B2_6130 - INFO [02-24|00:31:23.543] HTTP server started                      endpoint=127.0.0.1:6130 auth=false prefix= cors= vhosts=
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CBC80D3F2C70DE89BC14DF08CD90AF4D038A1AC46CA994A1C1DBAFEFF4BD46B2_6130 - INFO [02-24|00:31:23.577] New local node record                    seq=1,740,357,083,553 id=86e2edade17e6717 ip=10.1.0.33 udp=0 tcp=38303
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CBC80D3F2C70DE89BC14DF08CD90AF4D038A1AC46CA994A1C1DBAFEFF4BD46B2_6130 - INFO [02-24|00:31:23.579] New local node record                    seq=1,740,357,083,554 id=86e2edade17e6717 ip=10.1.0.33 udp=41540 tcp=38303
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CBC80D3F2C70DE89BC14DF08CD90AF4D038A1AC46CA994A1C1DBAFEFF4BD46B2_6130 - INFO [02-24|00:31:23.581] New local node record                    seq=1,740,357,083,555 id=86e2edade17e6717 ip=10.1.0.33 udp=41540 tcp=38303
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CBC80D3F2C70DE89BC14DF08CD90AF4D038A1AC46CA994A1C1DBAFEFF4BD46B2_6130 - INFO [02-24|00:31:23.583] New local node record                    seq=1,740,357,083,556 id=86e2edade17e6717 ip=10.1.0.33 udp=41540 tcp=38303
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CBC80D3F2C70DE89BC14DF08CD90AF4D038A1AC46CA994A1C1DBAFEFF4BD46B2_6130 - 2025-02-24T00:31:23.719Z	INFO	0x02cbc80d3f2c70de89bc14df08cd90af4d038a1ac46ca994a1c1dbafeff4bd46b2_6130	status-cli/util.go:62	existing account, key UID: 0xd8d0a6ec6b7198cb0217115fd766fab8ff146a351ef79446c2dec6d7dbf7a007
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CBC80D3F2C70DE89BC14DF08CD90AF4D038A1AC46CA994A1C1DBAFEFF4BD46B2_6130 - INFO [02-24|00:31:23.720] New local node record                    seq=1,740,357,083,557 id=86e2edade17e6717 ip=10.1.0.33 udp=41540 tcp=38303
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CBC80D3F2C70DE89BC14DF08CD90AF4D038A1AC46CA994A1C1DBAFEFF4BD46B2_6130 - 2025-02-24T00:31:23.742Z	INFO	0x02cbc80d3f2c70de89bc14df08cd90af4d038a1ac46ca994a1c1dbafeff4bd46b2_6130	status-cli/util.go:93	messenger started, public key: 0x04c61f8fa3515a136d6a0860b6cdb68ecd4b2d8df3f494a1b628b5f159c8168eb431422a84d88654a8bcffcccf7130e656351c12b7f31eb621de6d944d9c62b38d
[32mINFO    [0m src.node.status_node:status_node.py:52 Serving node with command: ['./status-cli', 'serve-account', '-n', '0x035cd90ef0acdd7ca1173984b56c36a1d9a0119d00f787c3b99bc9705b6d394094_6137', '-kid', '0x19f84f473f249eb2a536e729d1f2c0efd07cf8c214901a0faa4d83f2e71ece36']
[35mDEBUG   [0m src.libs.common:common.py:15 Sleeping for 2 seconds
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035CD90EF0ACDD7CA1173984B56C36A1D9A0119D00F787C3B99BC9705B6D394094_6137 - 2025-02-24T00:31:25.492Z	INFO	status-cli/serve.go:34	Running serve-account command, with:
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035CD90EF0ACDD7CA1173984B56C36A1D9A0119D00F787C3B99BC9705B6D394094_6137 - -name 0x035cd90ef0acdd7ca1173984b56c36a1d9a0119d00f787c3b99bc9705b6d394094_6137
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035CD90EF0ACDD7CA1173984B56C36A1D9A0119D00F787C3B99BC9705B6D394094_6137 - -key-uid 0x19f84f473f249eb2a536e729d1f2c0efd07cf8c214901a0faa4d83f2e71ece36
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035CD90EF0ACDD7CA1173984B56C36A1D9A0119D00F787C3B99BC9705B6D394094_6137 - -interactive false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035CD90EF0ACDD7CA1173984B56C36A1D9A0119D00F787C3B99BC9705B6D394094_6137 - -add
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035CD90EF0ACDD7CA1173984B56C36A1D9A0119D00F787C3B99BC9705B6D394094_6137 - -debug false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035CD90EF0ACDD7CA1173984B56C36A1D9A0119D00F787C3B99BC9705B6D394094_6137 - -help false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035CD90EF0ACDD7CA1173984B56C36A1D9A0119D00F787C3B99BC9705B6D394094_6137 - 
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035CD90EF0ACDD7CA1173984B56C36A1D9A0119D00F787C3B99BC9705B6D394094_6137 - 2025-02-24T00:31:25.492Z	INFO	0x035cd90ef0acdd7ca1173984b56c36a1d9a0119d00f787c3b99bc9705b6d394094_6137	status-cli/util.go:55	starting messenger
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035CD90EF0ACDD7CA1173984B56C36A1D9A0119D00F787C3B99BC9705B6D394094_6137 - INFO [02-24|00:31:25.536] Starting peer-to-peer node               instance=StatusIM/linux-amd64/go1.22.3
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035CD90EF0ACDD7CA1173984B56C36A1D9A0119D00F787C3B99BC9705B6D394094_6137 - INFO [02-24|00:31:25.542] New local node record                    seq=1,737,419,633,056 id=4d486e8dbc7eb7b6 ip=127.0.0.1 udp=0 tcp=37143
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035CD90EF0ACDD7CA1173984B56C36A1D9A0119D00F787C3B99BC9705B6D394094_6137 - INFO [02-24|00:31:25.542] Started P2P networking                   self="enode://88fdc7f7abca4ed0a72cfac7bbeb5ff6e6aeeb5dc2f2baf3f0e4dc9c1a9781862fa1544e09a59c772ded4335d9dbb9bc1e53eace961ebeb40bc9456f10dafa8f@127.0.0.1:37143?discport=0"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035CD90EF0ACDD7CA1173984B56C36A1D9A0119D00F787C3B99BC9705B6D394094_6137 - ERROR[02-24|00:31:25.545] Unavailable modules in HTTP API list     unavailable=[wakuv2] available="[admin debug web3 eth rpcstats appmetrics appgeneral peer personal status pending ens communitytokens stickers updates settings accounts multiaccounts browsers permissions mailservers gif chat wallet waku wakuext localnotifications]"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035CD90EF0ACDD7CA1173984B56C36A1D9A0119D00F787C3B99BC9705B6D394094_6137 - INFO [02-24|00:31:25.546] HTTP server started                      endpoint=127.0.0.1:6137 auth=false prefix= cors= vhosts=
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035CD90EF0ACDD7CA1173984B56C36A1D9A0119D00F787C3B99BC9705B6D394094_6137 - INFO [02-24|00:31:25.569] New local node record                    seq=1,740,357,085,550 id=16e46ba928397951 ip=10.1.0.33 udp=0 tcp=41371
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035CD90EF0ACDD7CA1173984B56C36A1D9A0119D00F787C3B99BC9705B6D394094_6137 - INFO [02-24|00:31:25.571] New local node record                    seq=1,740,357,085,551 id=16e46ba928397951 ip=10.1.0.33 udp=34458 tcp=41371
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035CD90EF0ACDD7CA1173984B56C36A1D9A0119D00F787C3B99BC9705B6D394094_6137 - INFO [02-24|00:31:25.574] New local node record                    seq=1,740,357,085,552 id=16e46ba928397951 ip=10.1.0.33 udp=34458 tcp=41371
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035CD90EF0ACDD7CA1173984B56C36A1D9A0119D00F787C3B99BC9705B6D394094_6137 - 2025-02-24T00:31:25.712Z	INFO	0x035cd90ef0acdd7ca1173984b56c36a1d9a0119d00f787c3b99bc9705b6d394094_6137	status-cli/util.go:62	existing account, key UID: 0x19f84f473f249eb2a536e729d1f2c0efd07cf8c214901a0faa4d83f2e71ece36
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035CD90EF0ACDD7CA1173984B56C36A1D9A0119D00F787C3B99BC9705B6D394094_6137 - INFO [02-24|00:31:25.715] New local node record                    seq=1,740,357,085,553 id=16e46ba928397951 ip=10.1.0.33 udp=34458 tcp=41371
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035CD90EF0ACDD7CA1173984B56C36A1D9A0119D00F787C3B99BC9705B6D394094_6137 - 2025-02-24T00:31:25.735Z	INFO	0x035cd90ef0acdd7ca1173984b56c36a1d9a0119d00f787c3b99bc9705b6d394094_6137	status-cli/util.go:93	messenger started, public key: 0x04f4a3da8d5133481b6c157534cf62275127bb04b5f4dd1fde013c583a2708bc40001ecc2ffbf4edcc4f03aab32c3f76de8ff673dc40d690ccda6d9dea3ada4f6c
[32mINFO    [0m src.node.status_node:status_node.py:52 Serving node with command: ['./status-cli', 'serve-account', '-n', '0x03084787b0eb1391fd622fda8fee5f9fda3571467ee2c4dbcce9d10e4cb50442ad_6133', '-kid', '0xa9461a9f60364bb61116eabe5cc9b95fd07f7c283d742e3ad11dfd6dbe9b5d89']
[35mDEBUG   [0m src.libs.common:common.py:15 Sleeping for 2 seconds
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - 2025-02-24T00:31:27.494Z	INFO	status-cli/serve.go:34	Running serve-account command, with:
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - -name 0x03084787b0eb1391fd622fda8fee5f9fda3571467ee2c4dbcce9d10e4cb50442ad_6133
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - -key-uid 0xa9461a9f60364bb61116eabe5cc9b95fd07f7c283d742e3ad11dfd6dbe9b5d89
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - -interactive false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - -add
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - -debug false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - -help false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - 
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - 2025-02-24T00:31:27.494Z	INFO	0x03084787b0eb1391fd622fda8fee5f9fda3571467ee2c4dbcce9d10e4cb50442ad_6133	status-cli/util.go:55	starting messenger
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - INFO [02-24|00:31:27.537] Starting peer-to-peer node               instance=StatusIM/linux-amd64/go1.22.3
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - ERROR[02-24|00:31:27.543] Unavailable modules in HTTP API list     unavailable=[wakuv2] available="[admin debug web3 eth rpcstats appmetrics appgeneral peer personal status pending ens communitytokens stickers updates settings accounts multiaccounts browsers permissions mailservers gif chat wallet waku wakuext localnotifications]"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - INFO [02-24|00:31:27.543] New local node record                    seq=1,737,419,596,535 id=ceb3ac46208d5874 ip=127.0.0.1 udp=0 tcp=38963
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - INFO [02-24|00:31:27.543] Started P2P networking                   self="enode://258928b145e8e2cfa71d9f79b1c6107f8a42182ee831449209e212c28895554f9555253596f26c14c2f2aebc9bab14e51fea9341444f9f3da43e3fb66230428b@127.0.0.1:38963?discport=0"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - INFO [02-24|00:31:27.544] HTTP server started                      endpoint=127.0.0.1:6133 auth=false prefix= cors= vhosts=
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - INFO [02-24|00:31:27.575] New local node record                    seq=1,740,357,087,558 id=a8112c1210b8aa7a ip=10.1.0.33 udp=0 tcp=43951
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - INFO [02-24|00:31:27.578] New local node record                    seq=1,740,357,087,559 id=a8112c1210b8aa7a ip=10.1.0.33 udp=43668 tcp=43951
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - INFO [02-24|00:31:27.585] New local node record                    seq=1,740,357,087,560 id=a8112c1210b8aa7a ip=10.1.0.33 udp=43668 tcp=43951
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - 2025-02-24T00:31:27.712Z	INFO	0x03084787b0eb1391fd622fda8fee5f9fda3571467ee2c4dbcce9d10e4cb50442ad_6133	status-cli/util.go:62	existing account, key UID: 0xa9461a9f60364bb61116eabe5cc9b95fd07f7c283d742e3ad11dfd6dbe9b5d89
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - INFO [02-24|00:31:27.713] New local node record                    seq=1,740,357,087,561 id=a8112c1210b8aa7a ip=10.1.0.33 udp=43668 tcp=43951
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - 2025-02-24T00:31:27.731Z	INFO	0x03084787b0eb1391fd622fda8fee5f9fda3571467ee2c4dbcce9d10e4cb50442ad_6133	status-cli/util.go:93	messenger started, public key: 0x040da8af244c48efedcce3e6d3c3ff3fa0059e8f99a00ee714878a2bd4596de3ce1a5d9869df28bd689db33753d4b39223a1a5d78d16b123c27c2c4d23a9e72f3e
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - INFO [02-24|00:31:27.742] New local node record                    seq=1,740,357,077,733 id=cd1aec2ad4860f22 ip=172.214.173.185 udp=40736 tcp=10176
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CBC80D3F2C70DE89BC14DF08CD90AF4D038A1AC46CA994A1C1DBAFEFF4BD46B2_6130 - 2025-02-24T00:31:28.743Z	INFO	0x02cbc80d3f2c70de89bc14df08cd90af4d038a1ac46ca994a1c1dbafeff4bd46b2_6130	status-cli/serve.go:107	retrieve messages...
[32mINFO    [0m src.node.status_node:status_node.py:52 Serving node with command: ['./status-cli', 'serve-account', '-n', '0x02d815e00131e404d7de053208ed6db77a996403243c52bf8b20922a5e947bbd43_6135', '-kid', '0x5991c117b01d9f460b63cae2ec74abb9afd2b85661af889425db7acf2047c393']
[35mDEBUG   [0m src.libs.common:common.py:15 Sleeping for 2 seconds
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02D815E00131E404D7DE053208ED6DB77A996403243C52BF8B20922A5E947BBD43_6135 - 2025-02-24T00:31:29.490Z	INFO	status-cli/serve.go:34	Running serve-account command, with:
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02D815E00131E404D7DE053208ED6DB77A996403243C52BF8B20922A5E947BBD43_6135 - -name 0x02d815e00131e404d7de053208ed6db77a996403243c52bf8b20922a5e947bbd43_6135
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02D815E00131E404D7DE053208ED6DB77A996403243C52BF8B20922A5E947BBD43_6135 - -key-uid 0x5991c117b01d9f460b63cae2ec74abb9afd2b85661af889425db7acf2047c393
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02D815E00131E404D7DE053208ED6DB77A996403243C52BF8B20922A5E947BBD43_6135 - -interactive false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02D815E00131E404D7DE053208ED6DB77A996403243C52BF8B20922A5E947BBD43_6135 - -add
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02D815E00131E404D7DE053208ED6DB77A996403243C52BF8B20922A5E947BBD43_6135 - -debug false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02D815E00131E404D7DE053208ED6DB77A996403243C52BF8B20922A5E947BBD43_6135 - -help false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02D815E00131E404D7DE053208ED6DB77A996403243C52BF8B20922A5E947BBD43_6135 - 
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02D815E00131E404D7DE053208ED6DB77A996403243C52BF8B20922A5E947BBD43_6135 - 2025-02-24T00:31:29.490Z	INFO	0x02d815e00131e404d7de053208ed6db77a996403243c52bf8b20922a5e947bbd43_6135	status-cli/util.go:55	starting messenger
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02D815E00131E404D7DE053208ED6DB77A996403243C52BF8B20922A5E947BBD43_6135 - INFO [02-24|00:31:29.530] Starting peer-to-peer node               instance=StatusIM/linux-amd64/go1.22.3
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02D815E00131E404D7DE053208ED6DB77A996403243C52BF8B20922A5E947BBD43_6135 - ERROR[02-24|00:31:29.535] Unavailable modules in HTTP API list     unavailable=[wakuv2] available="[admin debug web3 eth rpcstats appmetrics appgeneral peer personal status pending ens communitytokens stickers updates settings accounts multiaccounts browsers permissions mailservers gif chat wallet waku wakuext localnotifications]"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02D815E00131E404D7DE053208ED6DB77A996403243C52BF8B20922A5E947BBD43_6135 - INFO [02-24|00:31:29.535] New local node record                    seq=1,737,419,614,800 id=3120ffa1600d036e ip=127.0.0.1 udp=0 tcp=42987
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02D815E00131E404D7DE053208ED6DB77A996403243C52BF8B20922A5E947BBD43_6135 - INFO [02-24|00:31:29.536] Started P2P networking                   self="enode://1eee99e98b47a050566c4564713970fad84b9429798d00aea57ad2415987c1634a030183bee393717b626509c0bb811572a545295a0a6affb26c297bb7a1fbf9@127.0.0.1:42987?discport=0"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02D815E00131E404D7DE053208ED6DB77A996403243C52BF8B20922A5E947BBD43_6135 - INFO [02-24|00:31:29.536] HTTP server started                      endpoint=127.0.0.1:6135 auth=false prefix= cors= vhosts=
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02D815E00131E404D7DE053208ED6DB77A996403243C52BF8B20922A5E947BBD43_6135 - INFO [02-24|00:31:29.567] New local node record                    seq=1,740,357,089,547 id=2b69d29a72e09c89 ip=10.1.0.33 udp=0 tcp=40435
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02D815E00131E404D7DE053208ED6DB77A996403243C52BF8B20922A5E947BBD43_6135 - INFO [02-24|00:31:29.569] New local node record                    seq=1,740,357,089,548 id=2b69d29a72e09c89 ip=10.1.0.33 udp=51352 tcp=40435
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02D815E00131E404D7DE053208ED6DB77A996403243C52BF8B20922A5E947BBD43_6135 - INFO [02-24|00:31:29.570] New local node record                    seq=1,740,357,089,549 id=2b69d29a72e09c89 ip=10.1.0.33 udp=51352 tcp=40435
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02D815E00131E404D7DE053208ED6DB77A996403243C52BF8B20922A5E947BBD43_6135 - 2025-02-24T00:31:29.709Z	INFO	0x02d815e00131e404d7de053208ed6db77a996403243c52bf8b20922a5e947bbd43_6135	status-cli/util.go:62	existing account, key UID: 0x5991c117b01d9f460b63cae2ec74abb9afd2b85661af889425db7acf2047c393
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02D815E00131E404D7DE053208ED6DB77A996403243C52BF8B20922A5E947BBD43_6135 - INFO [02-24|00:31:29.710] New local node record                    seq=1,740,357,089,550 id=2b69d29a72e09c89 ip=10.1.0.33 udp=51352 tcp=40435
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02D815E00131E404D7DE053208ED6DB77A996403243C52BF8B20922A5E947BBD43_6135 - 2025-02-24T00:31:29.729Z	INFO	0x02d815e00131e404d7de053208ed6db77a996403243c52bf8b20922a5e947bbd43_6135	status-cli/util.go:93	messenger started, public key: 0x04890fe10a90ba0a415f5f05e7becd52b45f8f4c7c38236a549036dde17457d50811ab1bfb1b3144b4f859a3c323542ff810a716ef579e51035298bc110536cb3c
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035CD90EF0ACDD7CA1173984B56C36A1D9A0119D00F787C3B99BC9705B6D394094_6137 - 2025-02-24T00:31:30.735Z	INFO	0x035cd90ef0acdd7ca1173984b56c36a1d9a0119d00f787c3b99bc9705b6d394094_6137	status-cli/serve.go:107	retrieve messages...
[32mINFO    [0m src.node.status_node:status_node.py:52 Serving node with command: ['./status-cli', 'serve-account', '-n', '0x02e2daf7144be6e5fef22e6ea21f04302a51cba97ebe58c010478d8eac1aee62c6_6134', '-kid', '0xc74a6694789777559695ac9bf06c43cc94c5d4dd1cbd3c1d3eaabe55410855c4']
[35mDEBUG   [0m src.libs.common:common.py:15 Sleeping for 2 seconds
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02E2DAF7144BE6E5FEF22E6EA21F04302A51CBA97EBE58C010478D8EAC1AEE62C6_6134 - 2025-02-24T00:31:31.494Z	INFO	status-cli/serve.go:34	Running serve-account command, with:
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02E2DAF7144BE6E5FEF22E6EA21F04302A51CBA97EBE58C010478D8EAC1AEE62C6_6134 - -name 0x02e2daf7144be6e5fef22e6ea21f04302a51cba97ebe58c010478d8eac1aee62c6_6134
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02E2DAF7144BE6E5FEF22E6EA21F04302A51CBA97EBE58C010478D8EAC1AEE62C6_6134 - -key-uid 0xc74a6694789777559695ac9bf06c43cc94c5d4dd1cbd3c1d3eaabe55410855c4
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02E2DAF7144BE6E5FEF22E6EA21F04302A51CBA97EBE58C010478D8EAC1AEE62C6_6134 - -interactive false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02E2DAF7144BE6E5FEF22E6EA21F04302A51CBA97EBE58C010478D8EAC1AEE62C6_6134 - -add
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02E2DAF7144BE6E5FEF22E6EA21F04302A51CBA97EBE58C010478D8EAC1AEE62C6_6134 - -debug false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02E2DAF7144BE6E5FEF22E6EA21F04302A51CBA97EBE58C010478D8EAC1AEE62C6_6134 - -help false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02E2DAF7144BE6E5FEF22E6EA21F04302A51CBA97EBE58C010478D8EAC1AEE62C6_6134 - 
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02E2DAF7144BE6E5FEF22E6EA21F04302A51CBA97EBE58C010478D8EAC1AEE62C6_6134 - 2025-02-24T00:31:31.495Z	INFO	0x02e2daf7144be6e5fef22e6ea21f04302a51cba97ebe58c010478d8eac1aee62c6_6134	status-cli/util.go:55	starting messenger
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02E2DAF7144BE6E5FEF22E6EA21F04302A51CBA97EBE58C010478D8EAC1AEE62C6_6134 - INFO [02-24|00:31:31.535] Starting peer-to-peer node               instance=StatusIM/linux-amd64/go1.22.3
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02E2DAF7144BE6E5FEF22E6EA21F04302A51CBA97EBE58C010478D8EAC1AEE62C6_6134 - INFO [02-24|00:31:31.541] New local node record                    seq=1,737,419,605,674 id=1e19dde297f8874d ip=127.0.0.1 udp=0 tcp=44881
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02E2DAF7144BE6E5FEF22E6EA21F04302A51CBA97EBE58C010478D8EAC1AEE62C6_6134 - INFO [02-24|00:31:31.541] Started P2P networking                   self="enode://99a2b3d7a793500a85f73e4baddc4643c65467df907e6eb7d569a6434b36f4a3255fc0d41038ce07a88508f57eb8e3613c00a996639de8c6b65ea83232116fd0@127.0.0.1:44881?discport=0"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02E2DAF7144BE6E5FEF22E6EA21F04302A51CBA97EBE58C010478D8EAC1AEE62C6_6134 - ERROR[02-24|00:31:31.542] Unavailable modules in HTTP API list     unavailable=[wakuv2] available="[admin debug web3 eth rpcstats appmetrics appgeneral peer personal status pending ens communitytokens stickers updates settings accounts multiaccounts browsers permissions mailservers gif chat wallet waku wakuext localnotifications]"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02E2DAF7144BE6E5FEF22E6EA21F04302A51CBA97EBE58C010478D8EAC1AEE62C6_6134 - INFO [02-24|00:31:31.542] HTTP server started                      endpoint=127.0.0.1:6134 auth=false prefix= cors= vhosts=
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02E2DAF7144BE6E5FEF22E6EA21F04302A51CBA97EBE58C010478D8EAC1AEE62C6_6134 - INFO [02-24|00:31:31.577] New local node record                    seq=1,740,357,091,554 id=52db5313ff96284c ip=10.1.0.33 udp=49929 tcp=41839
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02E2DAF7144BE6E5FEF22E6EA21F04302A51CBA97EBE58C010478D8EAC1AEE62C6_6134 - INFO [02-24|00:31:31.579] New local node record                    seq=1,740,357,091,555 id=52db5313ff96284c ip=10.1.0.33 udp=49929 tcp=41839
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02E2DAF7144BE6E5FEF22E6EA21F04302A51CBA97EBE58C010478D8EAC1AEE62C6_6134 - 2025-02-24T00:31:31.720Z	INFO	0x02e2daf7144be6e5fef22e6ea21f04302a51cba97ebe58c010478d8eac1aee62c6_6134	status-cli/util.go:62	existing account, key UID: 0xc74a6694789777559695ac9bf06c43cc94c5d4dd1cbd3c1d3eaabe55410855c4
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02E2DAF7144BE6E5FEF22E6EA21F04302A51CBA97EBE58C010478D8EAC1AEE62C6_6134 - INFO [02-24|00:31:31.722] New local node record                    seq=1,740,357,091,556 id=52db5313ff96284c ip=10.1.0.33 udp=49929 tcp=41839
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02E2DAF7144BE6E5FEF22E6EA21F04302A51CBA97EBE58C010478D8EAC1AEE62C6_6134 - 2025-02-24T00:31:31.743Z	INFO	0x02e2daf7144be6e5fef22e6ea21f04302a51cba97ebe58c010478d8eac1aee62c6_6134	status-cli/util.go:93	messenger started, public key: 0x044da57de11adc9391ad5ca1677b0ff2da9e02f50f2184d27852966cb46e21a0759e1e0af6514a5d60dbec05dda5ab574f265a71a340caa3f98c92f9091f927a6d
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CBC80D3F2C70DE89BC14DF08CD90AF4D038A1AC46CA994A1C1DBAFEFF4BD46B2_6130 - WARN [02-24|00:31:31.750] Served personal_ecRecover                reqid=1 duration="68.227µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CBC80D3F2C70DE89BC14DF08CD90AF4D038A1AC46CA994A1C1DBAFEFF4BD46B2_6130 - WARN [02-24|00:31:31.752] Served personal_ecRecover                reqid=2 duration="35.966µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - 2025-02-24T00:31:32.731Z	INFO	0x03084787b0eb1391fd622fda8fee5f9fda3571467ee2c4dbcce9d10e4cb50442ad_6133	status-cli/serve.go:107	retrieve messages...
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CBC80D3F2C70DE89BC14DF08CD90AF4D038A1AC46CA994A1C1DBAFEFF4BD46B2_6130 - WARN [02-24|00:31:32.746] Served personal_ecRecover                reqid=3 duration="46.756µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CBC80D3F2C70DE89BC14DF08CD90AF4D038A1AC46CA994A1C1DBAFEFF4BD46B2_6130 - WARN [02-24|00:31:32.748] Served personal_ecRecover                reqid=4 duration="31.048µs" err="signature must be 65 bytes long"
[32mINFO    [0m src.node.status_node:status_node.py:52 Serving node with command: ['./status-cli', 'serve-account', '-n', '0x02c805d0bc6c60eae2df62a9780b0b46b75301283075057ecdc677a2dc7a3cbfe1_6139', '-kid', '0x98806521a617d0fb4d2140c292e22157c3d3197b212f81cd0a3bf164774580f0']
[35mDEBUG   [0m src.libs.common:common.py:15 Sleeping for 2 seconds
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02C805D0BC6C60EAE2DF62A9780B0B46B75301283075057ECDC677A2DC7A3CBFE1_6139 - 2025-02-24T00:31:33.500Z	INFO	status-cli/serve.go:34	Running serve-account command, with:
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02C805D0BC6C60EAE2DF62A9780B0B46B75301283075057ECDC677A2DC7A3CBFE1_6139 - -name 0x02c805d0bc6c60eae2df62a9780b0b46b75301283075057ecdc677a2dc7a3cbfe1_6139
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02C805D0BC6C60EAE2DF62A9780B0B46B75301283075057ECDC677A2DC7A3CBFE1_6139 - -key-uid 0x98806521a617d0fb4d2140c292e22157c3d3197b212f81cd0a3bf164774580f0
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02C805D0BC6C60EAE2DF62A9780B0B46B75301283075057ECDC677A2DC7A3CBFE1_6139 - -interactive false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02C805D0BC6C60EAE2DF62A9780B0B46B75301283075057ECDC677A2DC7A3CBFE1_6139 - -add
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02C805D0BC6C60EAE2DF62A9780B0B46B75301283075057ECDC677A2DC7A3CBFE1_6139 - -debug false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02C805D0BC6C60EAE2DF62A9780B0B46B75301283075057ECDC677A2DC7A3CBFE1_6139 - -help false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02C805D0BC6C60EAE2DF62A9780B0B46B75301283075057ECDC677A2DC7A3CBFE1_6139 - 
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02C805D0BC6C60EAE2DF62A9780B0B46B75301283075057ECDC677A2DC7A3CBFE1_6139 - 2025-02-24T00:31:33.500Z	INFO	0x02c805d0bc6c60eae2df62a9780b0b46b75301283075057ecdc677a2dc7a3cbfe1_6139	status-cli/util.go:55	starting messenger
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02C805D0BC6C60EAE2DF62A9780B0B46B75301283075057ECDC677A2DC7A3CBFE1_6139 - INFO [02-24|00:31:33.539] Starting peer-to-peer node               instance=StatusIM/linux-amd64/go1.22.3
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02C805D0BC6C60EAE2DF62A9780B0B46B75301283075057ECDC677A2DC7A3CBFE1_6139 - INFO [02-24|00:31:33.549] New local node record                    seq=1,737,419,651,311 id=acb6477d150fec3c ip=127.0.0.1 udp=0 tcp=40443
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02C805D0BC6C60EAE2DF62A9780B0B46B75301283075057ECDC677A2DC7A3CBFE1_6139 - INFO [02-24|00:31:33.550] Started P2P networking                   self="enode://437d152e7e8c57d86065f0879910c1f64ae6e14f007b9dc5b5c671ecfb463b66da4ffe7a8753f396f6232e58841a36808d6b65b1c05fb2a3aa9d8ed97ef447e0@127.0.0.1:40443?discport=0"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02C805D0BC6C60EAE2DF62A9780B0B46B75301283075057ECDC677A2DC7A3CBFE1_6139 - ERROR[02-24|00:31:33.550] Unavailable modules in HTTP API list     unavailable=[wakuv2] available="[admin debug web3 eth rpcstats appmetrics appgeneral peer personal status pending ens communitytokens stickers updates settings accounts multiaccounts browsers permissions mailservers gif chat wallet waku wakuext localnotifications]"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02C805D0BC6C60EAE2DF62A9780B0B46B75301283075057ECDC677A2DC7A3CBFE1_6139 - INFO [02-24|00:31:33.550] HTTP server started                      endpoint=127.0.0.1:6139 auth=false prefix= cors= vhosts=
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02C805D0BC6C60EAE2DF62A9780B0B46B75301283075057ECDC677A2DC7A3CBFE1_6139 - INFO [02-24|00:31:33.588] New local node record                    seq=1,740,357,093,563 id=0df0e70ab629e9be ip=10.1.0.33 udp=43119 tcp=43515
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02C805D0BC6C60EAE2DF62A9780B0B46B75301283075057ECDC677A2DC7A3CBFE1_6139 - INFO [02-24|00:31:33.590] New local node record                    seq=1,740,357,093,564 id=0df0e70ab629e9be ip=10.1.0.33 udp=43119 tcp=43515
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02C805D0BC6C60EAE2DF62A9780B0B46B75301283075057ECDC677A2DC7A3CBFE1_6139 - 2025-02-24T00:31:33.733Z	INFO	0x02c805d0bc6c60eae2df62a9780b0b46b75301283075057ecdc677a2dc7a3cbfe1_6139	status-cli/util.go:62	existing account, key UID: 0x98806521a617d0fb4d2140c292e22157c3d3197b212f81cd0a3bf164774580f0
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02C805D0BC6C60EAE2DF62A9780B0B46B75301283075057ECDC677A2DC7A3CBFE1_6139 - INFO [02-24|00:31:33.735] New local node record                    seq=1,740,357,093,565 id=0df0e70ab629e9be ip=10.1.0.33 udp=43119 tcp=43515
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02C805D0BC6C60EAE2DF62A9780B0B46B75301283075057ECDC677A2DC7A3CBFE1_6139 - 2025-02-24T00:31:33.755Z	INFO	0x02c805d0bc6c60eae2df62a9780b0b46b75301283075057ecdc677a2dc7a3cbfe1_6139	status-cli/util.go:93	messenger started, public key: 0x040d9cf92cec5a7f9e887525cd79bfc4a927af9e1ce7c4d89246d974613d6ea084c360b2bd11007d57e296de08a6fb0dd0d813d28e57cf5f797745241ddabdd973
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02D815E00131E404D7DE053208ED6DB77A996403243C52BF8B20922A5E947BBD43_6135 - 2025-02-24T00:31:34.729Z	INFO	0x02d815e00131e404d7de053208ed6db77a996403243c52bf8b20922a5e947bbd43_6135	status-cli/serve.go:107	retrieve messages...
[32mINFO    [0m src.node.status_node:status_node.py:52 Serving node with command: ['./status-cli', 'serve-account', '-n', '0x02195f3912bca1c5b6ea510d10a67f6c6aeba77702a48591a9c98240931603971c_6132', '-kid', '0x317d1096e362af7561ea2f494e3cf3f715137ba7da24d801eea4ac7e10895318']
[35mDEBUG   [0m src.libs.common:common.py:15 Sleeping for 2 seconds
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02195F3912BCA1C5B6EA510D10A67F6C6AEBA77702A48591A9C98240931603971C_6132 - 2025-02-24T00:31:35.504Z	INFO	status-cli/serve.go:34	Running serve-account command, with:
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02195F3912BCA1C5B6EA510D10A67F6C6AEBA77702A48591A9C98240931603971C_6132 - -name 0x02195f3912bca1c5b6ea510d10a67f6c6aeba77702a48591a9c98240931603971c_6132
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02195F3912BCA1C5B6EA510D10A67F6C6AEBA77702A48591A9C98240931603971C_6132 - -key-uid 0x317d1096e362af7561ea2f494e3cf3f715137ba7da24d801eea4ac7e10895318
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02195F3912BCA1C5B6EA510D10A67F6C6AEBA77702A48591A9C98240931603971C_6132 - -interactive false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02195F3912BCA1C5B6EA510D10A67F6C6AEBA77702A48591A9C98240931603971C_6132 - -add
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02195F3912BCA1C5B6EA510D10A67F6C6AEBA77702A48591A9C98240931603971C_6132 - -debug false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02195F3912BCA1C5B6EA510D10A67F6C6AEBA77702A48591A9C98240931603971C_6132 - -help false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02195F3912BCA1C5B6EA510D10A67F6C6AEBA77702A48591A9C98240931603971C_6132 - 
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02195F3912BCA1C5B6EA510D10A67F6C6AEBA77702A48591A9C98240931603971C_6132 - 2025-02-24T00:31:35.504Z	INFO	0x02195f3912bca1c5b6ea510d10a67f6c6aeba77702a48591a9c98240931603971c_6132	status-cli/util.go:55	starting messenger
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02195F3912BCA1C5B6EA510D10A67F6C6AEBA77702A48591A9C98240931603971C_6132 - INFO [02-24|00:31:35.549] Starting peer-to-peer node               instance=StatusIM/linux-amd64/go1.22.3
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02195F3912BCA1C5B6EA510D10A67F6C6AEBA77702A48591A9C98240931603971C_6132 - INFO [02-24|00:31:35.558] New local node record                    seq=1,737,419,587,417 id=a3e6d7f2cfadb3c4 ip=127.0.0.1 udp=0 tcp=45499
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02195F3912BCA1C5B6EA510D10A67F6C6AEBA77702A48591A9C98240931603971C_6132 - INFO [02-24|00:31:35.558] Started P2P networking                   self="enode://aad0dc447a381566b4aa3dc121ad9ba5674e454f820fcda769e16c748042a1e278d4c98d22a4e518f933d8135d897befd8ecfe65c0f92930c9606e794e24e381@127.0.0.1:45499?discport=0"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02195F3912BCA1C5B6EA510D10A67F6C6AEBA77702A48591A9C98240931603971C_6132 - ERROR[02-24|00:31:35.558] Unavailable modules in HTTP API list     unavailable=[wakuv2] available="[admin debug web3 eth rpcstats appmetrics appgeneral peer personal status pending ens communitytokens stickers updates settings accounts multiaccounts browsers permissions mailservers gif chat wallet waku wakuext localnotifications]"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02195F3912BCA1C5B6EA510D10A67F6C6AEBA77702A48591A9C98240931603971C_6132 - INFO [02-24|00:31:35.559] HTTP server started                      endpoint=127.0.0.1:6132 auth=false prefix= cors= vhosts=
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035CD90EF0ACDD7CA1173984B56C36A1D9A0119D00F787C3B99BC9705B6D394094_6137 - INFO [02-24|00:31:35.566] New local node record                    seq=1,740,357,085,554 id=16e46ba928397951 ip=172.214.173.185 udp=34458 tcp=10178
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02195F3912BCA1C5B6EA510D10A67F6C6AEBA77702A48591A9C98240931603971C_6132 - INFO [02-24|00:31:35.601] New local node record                    seq=1,740,357,095,572 id=3a3339a7ee500053 ip=10.1.0.33 udp=48602 tcp=43923
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02195F3912BCA1C5B6EA510D10A67F6C6AEBA77702A48591A9C98240931603971C_6132 - INFO [02-24|00:31:35.603] New local node record                    seq=1,740,357,095,573 id=3a3339a7ee500053 ip=10.1.0.33 udp=48602 tcp=43923
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02195F3912BCA1C5B6EA510D10A67F6C6AEBA77702A48591A9C98240931603971C_6132 - 2025-02-24T00:31:35.752Z	INFO	0x02195f3912bca1c5b6ea510d10a67f6c6aeba77702a48591a9c98240931603971c_6132	status-cli/util.go:62	existing account, key UID: 0x317d1096e362af7561ea2f494e3cf3f715137ba7da24d801eea4ac7e10895318
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02195F3912BCA1C5B6EA510D10A67F6C6AEBA77702A48591A9C98240931603971C_6132 - INFO [02-24|00:31:35.753] New local node record                    seq=1,740,357,095,574 id=3a3339a7ee500053 ip=10.1.0.33 udp=48602 tcp=43923
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02195F3912BCA1C5B6EA510D10A67F6C6AEBA77702A48591A9C98240931603971C_6132 - 2025-02-24T00:31:35.777Z	INFO	0x02195f3912bca1c5b6ea510d10a67f6c6aeba77702a48591a9c98240931603971c_6132	status-cli/util.go:93	messenger started, public key: 0x041389374bcaf9a1c36230942c58f4e3968c0a2472bfa9041916414f11d228a3b0605c6ede03ff65463fe8a22126d1f2c750991f03bdf352eeb6442611e2b87aa1
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02E2DAF7144BE6E5FEF22E6EA21F04302A51CBA97EBE58C010478D8EAC1AEE62C6_6134 - INFO [02-24|00:31:36.573] New local node record                    seq=1,740,357,091,557 id=52db5313ff96284c ip=172.214.173.185 udp=49929 tcp=10180
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02E2DAF7144BE6E5FEF22E6EA21F04302A51CBA97EBE58C010478D8EAC1AEE62C6_6134 - 2025-02-24T00:31:36.743Z	INFO	0x02e2daf7144be6e5fef22e6ea21f04302a51cba97ebe58c010478d8eac1aee62c6_6134	status-cli/serve.go:107	retrieve messages...
[32mINFO    [0m src.node.status_node:status_node.py:52 Serving node with command: ['./status-cli', 'serve-account', '-n', '0x021b95d970e2773934f546c0b698aaaf3e7754ab90cec96eb720b66fae1120421e_6138', '-kid', '0x1226dcff3cb70a51c0f10a4a47de02a888068b914e654e5626095edc6edf1707']
[35mDEBUG   [0m src.libs.common:common.py:15 Sleeping for 2 seconds
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X021B95D970E2773934F546C0B698AAAF3E7754AB90CEC96EB720B66FAE1120421E_6138 - 2025-02-24T00:31:37.502Z	INFO	status-cli/serve.go:34	Running serve-account command, with:
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X021B95D970E2773934F546C0B698AAAF3E7754AB90CEC96EB720B66FAE1120421E_6138 - -name 0x021b95d970e2773934f546c0b698aaaf3e7754ab90cec96eb720b66fae1120421e_6138
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X021B95D970E2773934F546C0B698AAAF3E7754AB90CEC96EB720B66FAE1120421E_6138 - -key-uid 0x1226dcff3cb70a51c0f10a4a47de02a888068b914e654e5626095edc6edf1707
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X021B95D970E2773934F546C0B698AAAF3E7754AB90CEC96EB720B66FAE1120421E_6138 - -interactive false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X021B95D970E2773934F546C0B698AAAF3E7754AB90CEC96EB720B66FAE1120421E_6138 - -add
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X021B95D970E2773934F546C0B698AAAF3E7754AB90CEC96EB720B66FAE1120421E_6138 - -debug false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X021B95D970E2773934F546C0B698AAAF3E7754AB90CEC96EB720B66FAE1120421E_6138 - -help false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X021B95D970E2773934F546C0B698AAAF3E7754AB90CEC96EB720B66FAE1120421E_6138 - 
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X021B95D970E2773934F546C0B698AAAF3E7754AB90CEC96EB720B66FAE1120421E_6138 - 2025-02-24T00:31:37.503Z	INFO	0x021b95d970e2773934f546c0b698aaaf3e7754ab90cec96eb720b66fae1120421e_6138	status-cli/util.go:55	starting messenger
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X021B95D970E2773934F546C0B698AAAF3E7754AB90CEC96EB720B66FAE1120421E_6138 - INFO [02-24|00:31:37.546] Starting peer-to-peer node               instance=StatusIM/linux-amd64/go1.22.3
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X021B95D970E2773934F546C0B698AAAF3E7754AB90CEC96EB720B66FAE1120421E_6138 - INFO [02-24|00:31:37.554] New local node record                    seq=1,737,419,642,180 id=e3228c0ebd0ad048 ip=127.0.0.1 udp=0 tcp=35129
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X021B95D970E2773934F546C0B698AAAF3E7754AB90CEC96EB720B66FAE1120421E_6138 - INFO [02-24|00:31:37.554] Started P2P networking                   self="enode://e1e8dc76505064a0cd4888bcc75868421543cdfbbcdbe271fd2f512ec4e0013e1e8829edf9dfcf895198aad4056c5e9c238c1c63eab627cfd4fa85b5908c1664@127.0.0.1:35129?discport=0"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X021B95D970E2773934F546C0B698AAAF3E7754AB90CEC96EB720B66FAE1120421E_6138 - ERROR[02-24|00:31:37.554] Unavailable modules in HTTP API list     unavailable=[wakuv2] available="[admin debug web3 eth rpcstats appmetrics appgeneral peer personal status pending ens communitytokens stickers updates settings accounts multiaccounts browsers permissions mailservers gif chat wallet waku wakuext localnotifications]"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X021B95D970E2773934F546C0B698AAAF3E7754AB90CEC96EB720B66FAE1120421E_6138 - INFO [02-24|00:31:37.554] HTTP server started                      endpoint=127.0.0.1:6138 auth=false prefix= cors= vhosts=
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - INFO [02-24|00:31:37.574] New local node record                    seq=1,740,357,087,562 id=a8112c1210b8aa7a ip=172.214.173.185 udp=43668 tcp=10178
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X021B95D970E2773934F546C0B698AAAF3E7754AB90CEC96EB720B66FAE1120421E_6138 - INFO [02-24|00:31:37.595] New local node record                    seq=1,740,357,097,570 id=6b9c3e304f64c353 ip=10.1.0.33 udp=0 tcp=37105
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X021B95D970E2773934F546C0B698AAAF3E7754AB90CEC96EB720B66FAE1120421E_6138 - INFO [02-24|00:31:37.596] New local node record                    seq=1,740,357,097,571 id=6b9c3e304f64c353 ip=10.1.0.33 udp=59347 tcp=37105
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X021B95D970E2773934F546C0B698AAAF3E7754AB90CEC96EB720B66FAE1120421E_6138 - INFO [02-24|00:31:37.598] New local node record                    seq=1,740,357,097,572 id=6b9c3e304f64c353 ip=10.1.0.33 udp=59347 tcp=37105
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X021B95D970E2773934F546C0B698AAAF3E7754AB90CEC96EB720B66FAE1120421E_6138 - 2025-02-24T00:31:37.746Z	INFO	0x021b95d970e2773934f546c0b698aaaf3e7754ab90cec96eb720b66fae1120421e_6138	status-cli/util.go:62	existing account, key UID: 0x1226dcff3cb70a51c0f10a4a47de02a888068b914e654e5626095edc6edf1707
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X021B95D970E2773934F546C0B698AAAF3E7754AB90CEC96EB720B66FAE1120421E_6138 - INFO [02-24|00:31:37.746] New local node record                    seq=1,740,357,097,573 id=6b9c3e304f64c353 ip=10.1.0.33 udp=59347 tcp=37105
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X021B95D970E2773934F546C0B698AAAF3E7754AB90CEC96EB720B66FAE1120421E_6138 - 2025-02-24T00:31:37.767Z	INFO	0x021b95d970e2773934f546c0b698aaaf3e7754ab90cec96eb720b66fae1120421e_6138	status-cli/util.go:93	messenger started, public key: 0x04c05ffc7fbe973116e8a7017bbd49632c7aaa31715924eb62497ce229bd52a6ffe6904b99dd534f66d4b1364770dce92fcc97a901a1adb98dbebddaf6bd876eb6
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CBC80D3F2C70DE89BC14DF08CD90AF4D038A1AC46CA994A1C1DBAFEFF4BD46B2_6130 - INFO [02-24|00:31:38.573] New local node record                    seq=1,740,357,083,558 id=86e2edade17e6717 ip=172.214.173.185 udp=41540 tcp=10176
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02C805D0BC6C60EAE2DF62A9780B0B46B75301283075057ECDC677A2DC7A3CBFE1_6139 - 2025-02-24T00:31:38.755Z	INFO	0x02c805d0bc6c60eae2df62a9780b0b46b75301283075057ecdc677a2dc7a3cbfe1_6139	status-cli/serve.go:107	retrieve messages...
[32mINFO    [0m src.node.status_node:status_node.py:52 Serving node with command: ['./status-cli', 'serve-account', '-n', '0x039e80917b3bfd4803fe7803ae300a62c373a7e0be767c16178980884a56208467_6136', '-kid', '0x81bb8c3cf7199471bd4b3dda97fca44a8d2a482f9cda5e413efb11fa68db3f52']
[35mDEBUG   [0m src.libs.common:common.py:15 Sleeping for 2 seconds
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X039E80917B3BFD4803FE7803AE300A62C373A7E0BE767C16178980884A56208467_6136 - 2025-02-24T00:31:39.506Z	INFO	status-cli/serve.go:34	Running serve-account command, with:
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X039E80917B3BFD4803FE7803AE300A62C373A7E0BE767C16178980884A56208467_6136 - -name 0x039e80917b3bfd4803fe7803ae300a62c373a7e0be767c16178980884a56208467_6136
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X039E80917B3BFD4803FE7803AE300A62C373A7E0BE767C16178980884A56208467_6136 - -key-uid 0x81bb8c3cf7199471bd4b3dda97fca44a8d2a482f9cda5e413efb11fa68db3f52
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X039E80917B3BFD4803FE7803AE300A62C373A7E0BE767C16178980884A56208467_6136 - -interactive false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X039E80917B3BFD4803FE7803AE300A62C373A7E0BE767C16178980884A56208467_6136 - -add
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X039E80917B3BFD4803FE7803AE300A62C373A7E0BE767C16178980884A56208467_6136 - -debug false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X039E80917B3BFD4803FE7803AE300A62C373A7E0BE767C16178980884A56208467_6136 - -help false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X039E80917B3BFD4803FE7803AE300A62C373A7E0BE767C16178980884A56208467_6136 - 
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X039E80917B3BFD4803FE7803AE300A62C373A7E0BE767C16178980884A56208467_6136 - 2025-02-24T00:31:39.506Z	INFO	0x039e80917b3bfd4803fe7803ae300a62c373a7e0be767c16178980884a56208467_6136	status-cli/util.go:55	starting messenger
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X039E80917B3BFD4803FE7803AE300A62C373A7E0BE767C16178980884A56208467_6136 - INFO [02-24|00:31:39.548] Starting peer-to-peer node               instance=StatusIM/linux-amd64/go1.22.3
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X039E80917B3BFD4803FE7803AE300A62C373A7E0BE767C16178980884A56208467_6136 - INFO [02-24|00:31:39.555] New local node record                    seq=1,737,419,623,946 id=877b25ac259fbb61 ip=127.0.0.1 udp=0 tcp=39485
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X039E80917B3BFD4803FE7803AE300A62C373A7E0BE767C16178980884A56208467_6136 - INFO [02-24|00:31:39.555] Started P2P networking                   self="enode://3c43eceef4f4dee8fd369a60b6400d45eefc85a84579959675a0d624fbbe868f4dbc522c79a78bf55c7844278981e3bd2bbe4ed39727170d2035215ea236eb0c@127.0.0.1:39485?discport=0"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X039E80917B3BFD4803FE7803AE300A62C373A7E0BE767C16178980884A56208467_6136 - ERROR[02-24|00:31:39.556] Unavailable modules in HTTP API list     unavailable=[wakuv2] available="[admin debug web3 eth rpcstats appmetrics appgeneral peer personal status pending ens communitytokens stickers updates settings accounts multiaccounts browsers permissions mailservers gif chat wallet waku wakuext localnotifications]"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X039E80917B3BFD4803FE7803AE300A62C373A7E0BE767C16178980884A56208467_6136 - INFO [02-24|00:31:39.556] HTTP server started                      endpoint=127.0.0.1:6136 auth=false prefix= cors= vhosts=
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02D815E00131E404D7DE053208ED6DB77A996403243C52BF8B20922A5E947BBD43_6135 - INFO [02-24|00:31:39.564] New local node record                    seq=1,740,357,089,551 id=2b69d29a72e09c89 ip=172.214.173.185 udp=51352 tcp=10179
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X039E80917B3BFD4803FE7803AE300A62C373A7E0BE767C16178980884A56208467_6136 - INFO [02-24|00:31:39.601] New local node record                    seq=1,740,357,099,565 id=0ed7003a408e2a65 ip=10.1.0.33 udp=0 tcp=40883
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X039E80917B3BFD4803FE7803AE300A62C373A7E0BE767C16178980884A56208467_6136 - INFO [02-24|00:31:39.603] New local node record                    seq=1,740,357,099,566 id=0ed7003a408e2a65 ip=10.1.0.33 udp=49231 tcp=40883
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X039E80917B3BFD4803FE7803AE300A62C373A7E0BE767C16178980884A56208467_6136 - INFO [02-24|00:31:39.605] New local node record                    seq=1,740,357,099,567 id=0ed7003a408e2a65 ip=10.1.0.33 udp=49231 tcp=40883
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X039E80917B3BFD4803FE7803AE300A62C373A7E0BE767C16178980884A56208467_6136 - 2025-02-24T00:31:39.749Z	INFO	0x039e80917b3bfd4803fe7803ae300a62c373a7e0be767c16178980884a56208467_6136	status-cli/util.go:62	existing account, key UID: 0x81bb8c3cf7199471bd4b3dda97fca44a8d2a482f9cda5e413efb11fa68db3f52
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X039E80917B3BFD4803FE7803AE300A62C373A7E0BE767C16178980884A56208467_6136 - INFO [02-24|00:31:39.750] New local node record                    seq=1,740,357,099,568 id=0ed7003a408e2a65 ip=10.1.0.33 udp=49231 tcp=40883
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X039E80917B3BFD4803FE7803AE300A62C373A7E0BE767C16178980884A56208467_6136 - 2025-02-24T00:31:39.776Z	INFO	0x039e80917b3bfd4803fe7803ae300a62c373a7e0be767c16178980884a56208467_6136	status-cli/util.go:93	messenger started, public key: 0x042898fe963497a6d5cdf4bae695090aae18993c81f12e39be2b51b3980b806fa22fd3688e25d44663ce4c02010d7d8856c7bff8c610d1c0d3be4f2d1187426bfe
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035CD90EF0ACDD7CA1173984B56C36A1D9A0119D00F787C3B99BC9705B6D394094_6137 - INFO [02-24|00:31:40.565] New local node record                    seq=1,740,357,085,555 id=16e46ba928397951 ip=172.214.173.185 udp=34458 tcp=10176
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02195F3912BCA1C5B6EA510D10A67F6C6AEBA77702A48591A9C98240931603971C_6132 - 2025-02-24T00:31:40.781Z	INFO	0x02195f3912bca1c5b6ea510d10a67f6c6aeba77702a48591a9c98240931603971c_6132	status-cli/serve.go:107	retrieve messages...
[32mINFO    [0m src.node.status_node:status_node.py:52 Serving node with command: ['./status-cli', 'serve-account', '-n', '0x036ddc1c91806eefcfbd8eb1accadc7eff82d79f1fed6e8fccd08f9cf737d02c1f_6131', '-kid', '0xe171a00735b54dcf9ecb7a27904fdd4b143362df5354c1e469753d5f08023459']
[35mDEBUG   [0m src.libs.common:common.py:15 Sleeping for 2 seconds
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036DDC1C91806EEFCFBD8EB1ACCADC7EFF82D79F1FED6E8FCCD08F9CF737D02C1F_6131 - 2025-02-24T00:31:41.507Z	INFO	status-cli/serve.go:34	Running serve-account command, with:
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036DDC1C91806EEFCFBD8EB1ACCADC7EFF82D79F1FED6E8FCCD08F9CF737D02C1F_6131 - -name 0x036ddc1c91806eefcfbd8eb1accadc7eff82d79f1fed6e8fccd08f9cf737d02c1f_6131
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036DDC1C91806EEFCFBD8EB1ACCADC7EFF82D79F1FED6E8FCCD08F9CF737D02C1F_6131 - -key-uid 0xe171a00735b54dcf9ecb7a27904fdd4b143362df5354c1e469753d5f08023459
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036DDC1C91806EEFCFBD8EB1ACCADC7EFF82D79F1FED6E8FCCD08F9CF737D02C1F_6131 - -interactive false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036DDC1C91806EEFCFBD8EB1ACCADC7EFF82D79F1FED6E8FCCD08F9CF737D02C1F_6131 - -add
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036DDC1C91806EEFCFBD8EB1ACCADC7EFF82D79F1FED6E8FCCD08F9CF737D02C1F_6131 - -debug false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036DDC1C91806EEFCFBD8EB1ACCADC7EFF82D79F1FED6E8FCCD08F9CF737D02C1F_6131 - -help false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036DDC1C91806EEFCFBD8EB1ACCADC7EFF82D79F1FED6E8FCCD08F9CF737D02C1F_6131 - 
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036DDC1C91806EEFCFBD8EB1ACCADC7EFF82D79F1FED6E8FCCD08F9CF737D02C1F_6131 - 2025-02-24T00:31:41.507Z	INFO	0x036ddc1c91806eefcfbd8eb1accadc7eff82d79f1fed6e8fccd08f9cf737d02c1f_6131	status-cli/util.go:55	starting messenger
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036DDC1C91806EEFCFBD8EB1ACCADC7EFF82D79F1FED6E8FCCD08F9CF737D02C1F_6131 - INFO [02-24|00:31:41.550] Starting peer-to-peer node               instance=StatusIM/linux-amd64/go1.22.3
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036DDC1C91806EEFCFBD8EB1ACCADC7EFF82D79F1FED6E8FCCD08F9CF737D02C1F_6131 - INFO [02-24|00:31:41.556] New local node record                    seq=1,737,419,578,301 id=91b1bc7236045679 ip=127.0.0.1 udp=0 tcp=35199
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036DDC1C91806EEFCFBD8EB1ACCADC7EFF82D79F1FED6E8FCCD08F9CF737D02C1F_6131 - INFO [02-24|00:31:41.556] Started P2P networking                   self="enode://91d8c4c0322908732547f0a21fe21a53e4a636758e1615a385e922f834ad499c271c69dcee9a8fefe3c7479c1584e71ead9adf547235b9547d0349db0b06bee9@127.0.0.1:35199?discport=0"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036DDC1C91806EEFCFBD8EB1ACCADC7EFF82D79F1FED6E8FCCD08F9CF737D02C1F_6131 - ERROR[02-24|00:31:41.556] Unavailable modules in HTTP API list     unavailable=[wakuv2] available="[admin debug web3 eth rpcstats appmetrics appgeneral peer personal status pending ens communitytokens stickers updates settings accounts multiaccounts browsers permissions mailservers gif chat wallet waku wakuext localnotifications]"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036DDC1C91806EEFCFBD8EB1ACCADC7EFF82D79F1FED6E8FCCD08F9CF737D02C1F_6131 - INFO [02-24|00:31:41.557] HTTP server started                      endpoint=127.0.0.1:6131 auth=false prefix= cors= vhosts=
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02E2DAF7144BE6E5FEF22E6EA21F04302A51CBA97EBE58C010478D8EAC1AEE62C6_6134 - INFO [02-24|00:31:41.573] New local node record                    seq=1,740,357,091,558 id=52db5313ff96284c ip=172.214.173.185 udp=49929 tcp=10180
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036DDC1C91806EEFCFBD8EB1ACCADC7EFF82D79F1FED6E8FCCD08F9CF737D02C1F_6131 - INFO [02-24|00:31:41.599] New local node record                    seq=1,740,357,101,576 id=e050a038e63efd27 ip=10.1.0.33 udp=53544 tcp=44533
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036DDC1C91806EEFCFBD8EB1ACCADC7EFF82D79F1FED6E8FCCD08F9CF737D02C1F_6131 - INFO [02-24|00:31:41.606] New local node record                    seq=1,740,357,101,577 id=e050a038e63efd27 ip=10.1.0.33 udp=53544 tcp=44533
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036DDC1C91806EEFCFBD8EB1ACCADC7EFF82D79F1FED6E8FCCD08F9CF737D02C1F_6131 - 2025-02-24T00:31:41.752Z	INFO	0x036ddc1c91806eefcfbd8eb1accadc7eff82d79f1fed6e8fccd08f9cf737d02c1f_6131	status-cli/util.go:62	existing account, key UID: 0xe171a00735b54dcf9ecb7a27904fdd4b143362df5354c1e469753d5f08023459
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036DDC1C91806EEFCFBD8EB1ACCADC7EFF82D79F1FED6E8FCCD08F9CF737D02C1F_6131 - INFO [02-24|00:31:41.754] New local node record                    seq=1,740,357,101,578 id=e050a038e63efd27 ip=10.1.0.33 udp=53544 tcp=44533
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036DDC1C91806EEFCFBD8EB1ACCADC7EFF82D79F1FED6E8FCCD08F9CF737D02C1F_6131 - 2025-02-24T00:31:41.783Z	INFO	0x036ddc1c91806eefcfbd8eb1accadc7eff82d79f1fed6e8fccd08f9cf737d02c1f_6131	status-cli/util.go:93	messenger started, public key: 0x047171e21124210e88fa7e404355815ceac9793d65706770d642813fdfaacc4e0def04dd42a5be6f28ef68d9e76a297cb7f6fe0326d50547faffaffbaf0fd1a662
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X021B95D970E2773934F546C0B698AAAF3E7754AB90CEC96EB720B66FAE1120421E_6138 - 2025-02-24T00:31:42.768Z	INFO	0x021b95d970e2773934f546c0b698aaaf3e7754ab90cec96eb720b66fae1120421e_6138	status-cli/serve.go:107	retrieve messages...
[35mDEBUG   [0m src.libs.common:common.py:15 Sleeping for 4 seconds
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02C805D0BC6C60EAE2DF62A9780B0B46B75301283075057ECDC677A2DC7A3CBFE1_6139 - INFO [02-24|00:31:43.583] New local node record                    seq=1,740,357,093,566 id=0df0e70ab629e9be ip=172.214.173.185 udp=43119 tcp=10182
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - WARN [02-24|00:31:43.904] Served personal_ecRecover                reqid=1 duration="47.769µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - WARN [02-24|00:31:43.905] Served personal_ecRecover                reqid=2 duration="32.701µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - WARN [02-24|00:31:43.907] Served personal_ecRecover                reqid=3 duration="30.868µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - WARN [02-24|00:31:43.909] Served personal_ecRecover                reqid=4 duration="29.164µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - 2025-02-24T00:31:43.920Z	INFO	0x03084787b0eb1391fd622fda8fee5f9fda3571467ee2c4dbcce9d10e4cb50442ad_6133	status-cli/serve.go:70	message received: d7f770e8-3a6c-44e7-aa6b-dfed4226bc6b_message_from_0x03084787b0eb1391fd622fda8fee5f9fda3571467ee2c4dbcce9d10e4cb50442ad_6133_10 (ID=0x96fd7737dbc87b6902628ed8818b8f568358ccf682c8c480a4e6461f302f5f6c)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - 2025-02-24T00:31:43.920Z	INFO	0x03084787b0eb1391fd622fda8fee5f9fda3571467ee2c4dbcce9d10e4cb50442ad_6133	status-cli/serve.go:70	message received: 3d6f729c-d4f0-4d88-ba2f-6d2499b2e642_message_from_0x03084787b0eb1391fd622fda8fee5f9fda3571467ee2c4dbcce9d10e4cb50442ad_6133_8 (ID=0x0cd0633bb374522a00803c1bf8f4b3dba732b0aa48f6881a4e4692e4c80cb90f)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - 2025-02-24T00:31:43.920Z	INFO	0x03084787b0eb1391fd622fda8fee5f9fda3571467ee2c4dbcce9d10e4cb50442ad_6133	status-cli/serve.go:70	message received: bb5a3ee0-6448-45e7-be7e-894eaa6ff575_message_from_0x03084787b0eb1391fd622fda8fee5f9fda3571467ee2c4dbcce9d10e4cb50442ad_6133_6 (ID=0x83a13a8f1978b6a8258e5283f0befb6d4c89667f6b13b585511749a7bef26cee)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - 2025-02-24T00:31:43.920Z	INFO	0x03084787b0eb1391fd622fda8fee5f9fda3571467ee2c4dbcce9d10e4cb50442ad_6133	status-cli/serve.go:70	message received: 3d4a9d32-b306-46a2-84e2-ffe900782d02_message_from_0x03084787b0eb1391fd622fda8fee5f9fda3571467ee2c4dbcce9d10e4cb50442ad_6133_2 (ID=0x60a5ebe76f35a72fb47364d71548f7ed9494419c95f221de39ccccc928e5115a)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - 2025-02-24T00:31:43.920Z	INFO	0x03084787b0eb1391fd622fda8fee5f9fda3571467ee2c4dbcce9d10e4cb50442ad_6133	status-cli/serve.go:70	message received: a8352bb4-f8d7-4deb-b626-af32e2fa480f_message_from_0x03084787b0eb1391fd622fda8fee5f9fda3571467ee2c4dbcce9d10e4cb50442ad_6133_24 (ID=0x35fc947e6394b5ad46abfcffc67a97bb7893b0f79afd968fc552289f87aa4fad)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - 2025-02-24T00:31:43.920Z	INFO	0x03084787b0eb1391fd622fda8fee5f9fda3571467ee2c4dbcce9d10e4cb50442ad_6133	status-cli/serve.go:70	message received: 5ae85cfc-1d66-4765-9299-438e410268e3_message_from_0x03084787b0eb1391fd622fda8fee5f9fda3571467ee2c4dbcce9d10e4cb50442ad_6133_18 (ID=0x19ef2c6f0ef2b96828df44da308b1e56966cc030465c479e101763e8fc979434)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - 2025-02-24T00:31:43.920Z	INFO	0x03084787b0eb1391fd622fda8fee5f9fda3571467ee2c4dbcce9d10e4cb50442ad_6133	status-cli/serve.go:70	message received: a7d0e964-1a75-4fc5-94a6-ed17f1643342_message_from_0x03084787b0eb1391fd622fda8fee5f9fda3571467ee2c4dbcce9d10e4cb50442ad_6133_16 (ID=0x5168e63d067f8faca93f5a16ae93a0fcd2a8d66d1b59a95d2ab880fe769840ab)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - 2025-02-24T00:31:43.920Z	INFO	0x03084787b0eb1391fd622fda8fee5f9fda3571467ee2c4dbcce9d10e4cb50442ad_6133	status-cli/serve.go:70	message received: 785480f7-8eea-43c5-8c0e-3323ca65c7d4_message_from_0x03084787b0eb1391fd622fda8fee5f9fda3571467ee2c4dbcce9d10e4cb50442ad_6133_12 (ID=0x79f9f2459be411908eba5eec25f49339896ace74aefcccdc234aaf9e9c916398)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - 2025-02-24T00:31:43.920Z	INFO	0x03084787b0eb1391fd622fda8fee5f9fda3571467ee2c4dbcce9d10e4cb50442ad_6133	status-cli/serve.go:70	message received: 6d9a3ad5-cb77-4a96-8e60-f209233de125_message_from_0x03084787b0eb1391fd622fda8fee5f9fda3571467ee2c4dbcce9d10e4cb50442ad_6133_0 (ID=0x422d6f6b9d3359e3ae6eef47c8b4453af35c4c922dd867a19ca8490871bd91ae)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - 2025-02-24T00:31:43.920Z	INFO	0x03084787b0eb1391fd622fda8fee5f9fda3571467ee2c4dbcce9d10e4cb50442ad_6133	status-cli/serve.go:70	message received: 3e9015bd-2374-4c63-8732-cc889d01708f_message_from_0x03084787b0eb1391fd622fda8fee5f9fda3571467ee2c4dbcce9d10e4cb50442ad_6133_22 (ID=0x417054bfe350bc8d04fe597e72a5c397ad5a6f2530f079337712f27619f4dcb3)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - 2025-02-24T00:31:43.920Z	INFO	0x03084787b0eb1391fd622fda8fee5f9fda3571467ee2c4dbcce9d10e4cb50442ad_6133	status-cli/serve.go:70	message received: d4285dd0-05e4-44d7-bf55-6cfb7a4431d7_message_from_0x03084787b0eb1391fd622fda8fee5f9fda3571467ee2c4dbcce9d10e4cb50442ad_6133_20 (ID=0xf0bd821a5cc37f40a9192955022bf21ae3e8972c437ec0def006b564e076acd1)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - 2025-02-24T00:31:43.920Z	INFO	0x03084787b0eb1391fd622fda8fee5f9fda3571467ee2c4dbcce9d10e4cb50442ad_6133	status-cli/serve.go:70	message received: 6e5d5f3e-2cfd-46de-8fac-58f8836ffa28_message_from_0x03084787b0eb1391fd622fda8fee5f9fda3571467ee2c4dbcce9d10e4cb50442ad_6133_14 (ID=0xf014a62c861cf7085a4ab9bc381bfcf86be35c295e62134e83c97a74158e31ef)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - 2025-02-24T00:31:43.920Z	INFO	0x03084787b0eb1391fd622fda8fee5f9fda3571467ee2c4dbcce9d10e4cb50442ad_6133	status-cli/serve.go:70	message received: 6b97bed0-7ec0-45c0-9fe6-c4bb2bb391b2_message_from_0x03084787b0eb1391fd622fda8fee5f9fda3571467ee2c4dbcce9d10e4cb50442ad_6133_4 (ID=0x440b1c70584e5bf2149e74a7e943832c8f38b2fca3138bb70e284627a0e70f9a)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02D815E00131E404D7DE053208ED6DB77A996403243C52BF8B20922A5E947BBD43_6135 - INFO [02-24|00:31:44.563] New local node record                    seq=1,740,357,089,552 id=2b69d29a72e09c89 ip=172.214.173.185 udp=51352 tcp=10179
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X039E80917B3BFD4803FE7803AE300A62C373A7E0BE767C16178980884A56208467_6136 - 2025-02-24T00:31:44.776Z	INFO	0x039e80917b3bfd4803fe7803ae300a62c373a7e0be767c16178980884a56208467_6136	status-cli/serve.go:107	retrieve messages...
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02195F3912BCA1C5B6EA510D10A67F6C6AEBA77702A48591A9C98240931603971C_6132 - INFO [02-24|00:31:45.593] New local node record                    seq=1,740,357,095,575 id=3a3339a7ee500053 ip=172.214.173.185 udp=48602 tcp=10183
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02E2DAF7144BE6E5FEF22E6EA21F04302A51CBA97EBE58C010478D8EAC1AEE62C6_6134 - INFO [02-24|00:31:46.572] New local node record                    seq=1,740,357,091,559 id=52db5313ff96284c ip=172.214.173.185 udp=49929 tcp=10181
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036DDC1C91806EEFCFBD8EB1ACCADC7EFF82D79F1FED6E8FCCD08F9CF737D02C1F_6131 - 2025-02-24T00:31:46.784Z	INFO	0x036ddc1c91806eefcfbd8eb1accadc7eff82d79f1fed6e8fccd08f9cf737d02c1f_6131	status-cli/serve.go:107	retrieve messages...
[35mDEBUG   [0m src.steps.common:common.py:49 Entering context manager: add_packet_loss
[35mDEBUG   [0m src.node.rpc_client:rpc_client.py:21 Node: first_node sends request at address: http://127.0.0.1:29639 with payload: {"jsonrpc": "2.0", "method": "wakuext_fetchCommunity", "params": [{"communityKey": "0x02cbc80d3f2c70de89bc14df08cd90af4d038a1ac46ca994a1c1dbafeff4bd46b2", "waitForResponse": true, "tryDatabase": true}], "id": 1}
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X021B95D970E2773934F546C0B698AAAF3E7754AB90CEC96EB720B66FAE1120421E_6138 - INFO [02-24|00:31:47.591] New local node record                    seq=1,740,357,097,574 id=6b9c3e304f64c353 ip=172.214.173.185 udp=59347 tcp=10184
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036DDC1C91806EEFCFBD8EB1ACCADC7EFF82D79F1FED6E8FCCD08F9CF737D02C1F_6131 - INFO [02-24|00:31:56.593] New local node record                    seq=1,740,357,101,579 id=e050a038e63efd27 ip=172.214.173.185 udp=53544 tcp=10185
[31m[1mERROR   [0m src.node.rpc_client:rpc_client.py:32 Request to http://127.0.0.1:29639 usig method wakuext_fetchCommunity timed out after 10 seconds. Node: first_node
[35mDEBUG   [0m src.node.rpc_client:rpc_client.py:21 Node: first_node sends request at address: http://127.0.0.1:29639 with payload: {"jsonrpc": "2.0", "method": "wakuext_fetchCommunity", "params": [{"communityKey": "0x02cbc80d3f2c70de89bc14df08cd90af4d038a1ac46ca994a1c1dbafeff4bd46b2", "waitForResponse": true, "tryDatabase": true}], "id": 1}
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X021B95D970E2773934F546C0B698AAAF3E7754AB90CEC96EB720B66FAE1120421E_6138 - INFO [02-24|00:32:02.590] New local node record                    seq=1,740,357,097,575 id=6b9c3e304f64c353 ip=172.214.173.185 udp=59347 tcp=10183
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X039E80917B3BFD4803FE7803AE300A62C373A7E0BE767C16178980884A56208467_6136 - INFO [02-24|00:32:04.594] New local node record                    seq=1,740,357,099,569 id=0ed7003a408e2a65 ip=172.214.173.185 udp=49231 tcp=10184
[31m[1mERROR   [0m src.node.rpc_client:rpc_client.py:32 Request to http://127.0.0.1:29639 usig method wakuext_fetchCommunity timed out after 10 seconds. Node: first_node
[35mDEBUG   [0m src.node.rpc_client:rpc_client.py:21 Node: first_node sends request at address: http://127.0.0.1:29639 with payload: {"jsonrpc": "2.0", "method": "wakuext_fetchCommunity", "params": [{"communityKey": "0x02cbc80d3f2c70de89bc14df08cd90af4d038a1ac46ca994a1c1dbafeff4bd46b2", "waitForResponse": true, "tryDatabase": true}], "id": 1}
[31m[1mERROR   [0m src.node.rpc_client:rpc_client.py:32 Request to http://127.0.0.1:29639 usig method wakuext_fetchCommunity timed out after 10 seconds. Node: first_node
[35mDEBUG   [0m src.steps.common:common.py:54 Exiting context manager: add_packet_loss
[35mDEBUG   [0m tests.conftest:conftest.py:42 Running fixture teardown: save_node_logs_on_fail
[35mDEBUG   [0m tests.conftest:conftest.py:45 Test failed, attempting to rename and move logs to the test_results folder
[35mDEBUG   [0m tests.conftest:conftest.py:55 Moved log file 0x035cd90ef0acdd7ca1173984b56c36a1d9a0119d00f787c3b99bc9705b6d394094_6137.log to test_results/testjoincommunity_test_join_community_with_packet_loss_0x035cd90ef0acdd7ca1173984b56c36a1d9a0119d00f787c3b99bc9705b6d394094_6137.log
[35mDEBUG   [0m tests.conftest:conftest.py:55 Moved log file 0x039e80917b3bfd4803fe7803ae300a62c373a7e0be767c16178980884a56208467_6136.log to test_results/testjoincommunity_test_join_community_with_packet_loss_0x039e80917b3bfd4803fe7803ae300a62c373a7e0be767c16178980884a56208467_6136.log
[35mDEBUG   [0m tests.conftest:conftest.py:55 Moved log file 0x02d815e00131e404d7de053208ed6db77a996403243c52bf8b20922a5e947bbd43_6135.log to test_results/testjoincommunity_test_join_community_with_packet_loss_0x02d815e00131e404d7de053208ed6db77a996403243c52bf8b20922a5e947bbd43_6135.log
[35mDEBUG   [0m tests.conftest:conftest.py:55 Moved log file first_node.log to test_results/testjoincommunity_test_join_community_with_packet_loss_first_node.log
[35mDEBUG   [0m tests.conftest:conftest.py:55 Moved log file 0x021b95d970e2773934f546c0b698aaaf3e7754ab90cec96eb720b66fae1120421e_6138.log to test_results/testjoincommunity_test_join_community_with_packet_loss_0x021b95d970e2773934f546c0b698aaaf3e7754ab90cec96eb720b66fae1120421e_6138.log
[35mDEBUG   [0m tests.conftest:conftest.py:55 Moved log file 0x02cbc80d3f2c70de89bc14df08cd90af4d038a1ac46ca994a1c1dbafeff4bd46b2_6130.log to test_results/testjoincommunity_test_join_community_with_packet_loss_0x02cbc80d3f2c70de89bc14df08cd90af4d038a1ac46ca994a1c1dbafeff4bd46b2_6130.log
[35mDEBUG   [0m tests.conftest:conftest.py:55 Moved log file 0x03084787b0eb1391fd622fda8fee5f9fda3571467ee2c4dbcce9d10e4cb50442ad_6133.log to test_results/testjoincommunity_test_join_community_with_packet_loss_0x03084787b0eb1391fd622fda8fee5f9fda3571467ee2c4dbcce9d10e4cb50442ad_6133.log
[35mDEBUG   [0m tests.conftest:conftest.py:55 Moved log file 0x02195f3912bca1c5b6ea510d10a67f6c6aeba77702a48591a9c98240931603971c_6132.log to test_results/testjoincommunity_test_join_community_with_packet_loss_0x02195f3912bca1c5b6ea510d10a67f6c6aeba77702a48591a9c98240931603971c_6132.log
[35mDEBUG   [0m tests.conftest:conftest.py:55 Moved log file 0x036ddc1c91806eefcfbd8eb1accadc7eff82d79f1fed6e8fccd08f9cf737d02c1f_6131.log to test_results/testjoincommunity_test_join_community_with_packet_loss_0x036ddc1c91806eefcfbd8eb1accadc7eff82d79f1fed6e8fccd08f9cf737d02c1f_6131.log
[35mDEBUG   [0m tests.conftest:conftest.py:55 Moved log file 0x02c805d0bc6c60eae2df62a9780b0b46b75301283075057ecdc677a2dc7a3cbfe1_6139.log to test_results/testjoincommunity_test_join_community_with_packet_loss_0x02c805d0bc6c60eae2df62a9780b0b46b75301283075057ecdc677a2dc7a3cbfe1_6139.log
[35mDEBUG   [0m tests.conftest:conftest.py:55 Moved log file 0x02e2daf7144be6e5fef22e6ea21f04302a51cba97ebe58c010478d8eac1aee62c6_6134.log to test_results/testjoincommunity_test_join_community_with_packet_loss_0x02e2daf7144be6e5fef22e6ea21f04302a51cba97ebe58c010478d8eac1aee62c6_6134.log
[35mDEBUG   [0m tests.conftest:conftest.py:62 Running fixture teardown: clear_open_nodes
[32mINFO    [0m src.node.status_node:status_node.py:73 Stopping node with name: first_node
[32mINFO    [0m src.node.status_node:status_node.py:73 Stopping node with name: 0x02cbc80d3f2c70de89bc14df08cd90af4d038a1ac46ca994a1c1dbafeff4bd46b2_6130
[32mINFO    [0m src.node.status_node:status_node.py:73 Stopping node with name: 0x035cd90ef0acdd7ca1173984b56c36a1d9a0119d00f787c3b99bc9705b6d394094_6137
[32mINFO    [0m src.node.status_node:status_node.py:73 Stopping node with name: 0x03084787b0eb1391fd622fda8fee5f9fda3571467ee2c4dbcce9d10e4cb50442ad_6133
[32mINFO    [0m src.node.status_node:status_node.py:73 Stopping node with name: 0x02d815e00131e404d7de053208ed6db77a996403243c52bf8b20922a5e947bbd43_6135
[32mINFO    [0m src.node.status_node:status_node.py:73 Stopping node with name: 0x02e2daf7144be6e5fef22e6ea21f04302a51cba97ebe58c010478d8eac1aee62c6_6134
[32mINFO    [0m src.node.status_node:status_node.py:73 Stopping node with name: 0x02c805d0bc6c60eae2df62a9780b0b46b75301283075057ecdc677a2dc7a3cbfe1_6139
[32mINFO    [0m src.node.status_node:status_node.py:73 Stopping node with name: 0x02195f3912bca1c5b6ea510d10a67f6c6aeba77702a48591a9c98240931603971c_6132
[32mINFO    [0m src.node.status_node:status_node.py:73 Stopping node with name: 0x021b95d970e2773934f546c0b698aaaf3e7754ab90cec96eb720b66fae1120421e_6138
[32mINFO    [0m src.node.status_node:status_node.py:73 Stopping node with name: 0x039e80917b3bfd4803fe7803ae300a62c373a7e0be767c16178980884a56208467_6136
[32mINFO    [0m src.node.status_node:status_node.py:73 Stopping node with name: 0x036ddc1c91806eefcfbd8eb1accadc7eff82d79f1fed6e8fccd08f9cf737d02c1f_6131